spk/python/src/service-setup.sh

```diff
@@ -0,0 +1,17 @@
+PATH="${SYNOPKG_PKGDEST}/bin:${PATH}"
+
+service_postinst ()
+{
+    # Install the wheels
+    ${SYNOPKG_PKGDEST}/bin/pip install --no-deps --no-index -U --force-reinstall -f ${SYNOPKG_PKGDEST}/share/wheelhouse ${SYNOPKG_PKGDEST}/share/wheelhouse/*.whl >> ${INST_LOG}
```

Comment:
I would add stderr redirection to file too with `2>&1` (here and in following lines)